cli/src/main.rs

```diff
@@ -0,0 +1,125 @@
+mod fstools;
+mod location;
+mod modules;
+mod spawner;
+mod target;
+mod tools;
+mod tracker;
+
+use clap::{Parser, Subcommand};
+use futures::future::join_all;
+use location::Location;
+use modules::Manager;
+use std::io::{Error, ErrorKind};
+use target::Target;
+use tools::RemoveDuplicates;
+use tracker::Tracker;
+
+#[macro_use]
+extern crate lazy_static;
+
+lazy_static! {
+    static ref LOCATION: Location = Location::new().expect("Fail to setup location of ICSMW");
+    static ref TRACKER: Tracker = Tracker::new();
+}
```

Comment:
I took a look on the current progress in rust standard library of [OnceLock](https://doc.rust-lang.org/std/sync/struct.OnceLock.html) and [LazyLock](https://doc.rust-lang.org/std/cell/struct.LazyCell.html).
Since LazyLock is in nightly only, I don't think that we need to change the current implementation